apis/components/v1/kserve_types.go

```diff
@@ -21,6 +21,10 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	ModelMeshCtrlerComponentName = "odh-model-controller" // shared by kserve and mm
```

Comment:
```suggestion
	ModelControllerComponentName = "odh-model-controller" // shared by kserve and mm
```

I assume that it shouldn't have ModelMesh in the name if it's not modelmesh specific?